internal/prompt/prompt.go

```diff
@@ -252,10 +205,13 @@ func Seed(reader *bufio.Reader) (seed []byte, imported bool, err error) {
 			return nil, false, err
 		}
 
-		seedStrSplit := walletseed.EncodeMnemonicSlice(seed)
+		seedStrSplit, err := walletseed.EncodeMnemonicSlice(seed)
+		if err != nil {
+			return nil, false, err
+		}
 
 		fmt.Println("Your wallet generation seed is:")
-		for i := 0; i < hdkeychain.RecommendedSeedLen+1; i++ {
+		for i := 0; i < len(seedStrSplit); i++ {
 			fmt.Printf("%v ", seedStrSplit[i])
```

Comment:
Is this for debugging? :) If so then we should use a logger with debug level so it only prints when debugging

Reply:
No, this is `prompt.go` file that handles interactive CLI commands. Printouts from this tool should be displayed as is, without timestamps, log levels etc. that are typical for loggers.